### ISSUE TITLE
incoherency on data about EPS in endpoints

### ISSUE THREAD

**Issue by muziofg** [antoinevulcain/Financial-Modeling-Prep-API#24]
Example: AMD 

eps (the sum of last four, I think) reported here are updated (I think): https://financialmodelingprep.com/api/v3/quote/AMD
eps reported here not: https://financialmodelingprep.com/api/v3.1/financials/income-statement/AMD?period=quarter

**Comment by antoinevulcain** [comment]
Good point, one come from the SEC us-gaap the other is calculated, will look into it. thanks for pointing this out!
